docs/usage/configuration/config_documentation.md

```diff
@@ -3398,8 +3398,8 @@ This option has the following sub-options:
     rebuild the indexes in order to search through all known users.
     These indexes are built the first time Synapse starts; admins can
     manually trigger a rebuild via API following the instructions at
-         https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/background_updates.html#run
-    Set to true to return search results containing all known users, even if that
+         [https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/background_updates.html#run](https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/background_updates.html#run),
```

Comment:
In markdown you can convert a URI to a link by wrapping it in `<...>`s. So you could write:

```
<https://matrix-org.github.io/synapse/latest/usage/administration/admin_api/background_updates.html#run>
```

However, it would be nice to use some words as the link text instead. This ensure the documentation does not need to be changed if we switched URL's for example (or someone else decides to self-host it).

So I'd instead write something like:

```
following the instructions [for running background updates](../administration/admin_api/background_updates.md#run). Set to true ...
```